ui.content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -95,7 +95,7 @@
             pattern="/etc/acs-commons/reports(.*)?"
             mode="merge"/>
```

Comment:
also mode=merge is only supported on filter elements,  I think line 94ff can be removed

Reply:
Unrelated to this card.